tests/azul_agent.py

```diff
@@ -12,6 +12,11 @@ def __init__(self, deployment):
     def get_specimen_by_project(self, project_shortname):
         filters = {"file": {"project": {"is": [project_shortname]}}}
         path = '/repository/specimens?filters=' + urllib.parse.quote(json.dumps(filters))
-        response = requests.get(self.azul_url + path)
+        azul_session = requests.Session()
```

Comment:
Ideally, I'd want to stash the session in an instance attribute.

Reply:
And I'd call the attribute just `session` or `requests_session` or, actually, `https_session`.